### PR TITLE
[Korean] Enhance and fix problems

### PR DESCRIPTION
**Pull request by crowjdh**
Fix some typos, wrong translations, misplaced code blocks and enhanced few readability issues.

This pr has several commits, so feel free to perform squash and merge.

### CODE REVIEW

**Comment by aniav** [comment]
@DjangoGirls/translation-managers do we have someone to review korean? :)

**Comment by aniav** [comment]
@mallina would you be willing to review these changes? And maybe you would have time to become a korean translation reviewer? :)

**Comment by aniav** [comment]
@crowjdh I am sorry this is taking so long. Maybe you have someone who would be willing to review your changes? We don't have anyone for the Korean translations for now :(

**Comment by mallina** [comment]
!!! Sorry, I completely missed this email. Right on it

File: ko/django_forms/README.md [hunk 1]
```diff
@@ -336,14 +332,14 @@ def post_edit(request, pk):
 
 음…. 코드가 `post_new`와 거의 비슷해 보이지 않나요? 하지만 완전히 같지는 않아요.
 * 첫 번째: url로부터 추가로 `pk` 매개변수를 받아서 처리합니다.
-* 두 번째: `get_object_or_404(Post, pk=pk)`를 호출하여 수정하고자 하는 글의 `Post` 모델 `인스턴스(instance)`로 가져옵니다. (`pk`로 원하는 글을 찾습니다) 이렇게 가져온 데이터를 폼을 만들 때와(글을 수정할 때 폼에 이전에 입력했던 데이터가 있어야 하겠죠?) 폼을 저장할 때 사용하게 됩니다.
+* 두 번째: `get_object_or_404(Post, pk=pk)`를 호출하여 수정하고자 하는 글의 `Post` 모델 `인스턴스(instance)`를 가져온 후, 아래와 같이 폼을 저장할 때...
```
Comment (soomin-jeong) [inline ko/django_forms/README.md]: The conversion of the previous version and the new one would be the best : " 두 번째: `get_object_or_404(Post, pk=pk)`를 호출하여 수정하고자 하는 글의 `Post` 모델 `인스턴스(instance)`를 가져온 후, 아래와 같이 폼을 저장할 때 사용하게 됩니다. " on line 335.

File: ko/django_forms/README.md [hunk 2]
```diff
@@ -336,14 +332,14 @@ def post_edit(request, pk):
...
 
 {% filename %}blog/views.py{% endfilename %}
 ```python
 form = PostForm(request.POST, instance=post)
 ```
 
-그리고 폼에 아래와 같이 수정하세요.
+...와 `post`를 사용해 수정할 폼을 열 때 사용하게 됩니다.
```
Comment (soomin-jeong) [inline ko/django_forms/README.md]: Slightly different from the original statement. I would say: "그리고 다음과 같이 수정할 포스트를 가지고 폼을 열 때 사용할 수 있습니다."

File: ko/django_forms/README.md [hunk 3]
```diff
@@ -389,7 +385,7 @@ https://docs.djangoproject.com/en/1.10/topics/forms/
 
 세부 페이지에 있는 수정 아이콘이 기억나죠? 이번에도 동일하게 다른 사람들이 게시글을 수정하지 못하게 할 거에요.
 
-`blog/templates/blog/post_detail.html`파일을 열어 아래 내용을 추가하세요.
+`blog/templates/blog/post_detail.html`파일을 연 후 아래 라인을 찾아서:
```
Comment (soomin-jeong) [inline ko/django_forms/README.md]: Original version seems fine.

File: ko/django_forms/README.md [hunk 4]
```diff
@@ -258,7 +254,7 @@ def post_new(request):
 
 모두 잘해냈어요!
 
-> 장고 관리자 인터페이스처럼 로그인된 상태라고 생각해봅시다. 하지만 사용자가 로그아웃되는 상황이 발생하기도 하죠. (브라우저가 닫히거나, DB가 재시작된다든가 등) 만약 로그인되지 않은 상태에서 새 글을 저장한다면, 사용자가 로그인되어있지 않아 누가 글을 작성하였는지 알 수 없어요. 그래서 글을 저장할 때 오류가 발생하고, 로그인시키도록 [http://127.0.0.1:8000/admin](http://127.0.0.1:8000/admin) 관리자 페이지가 나타나게 될 거에요. 이 문제는 금방 해결할 수 있어요. 튜토리얼을 마친 후 장고걸스 심화 튜토리얼 - 보안 추가하기 장에서 실습할 수 있어요.
+> 장고 관리자 인터페이스처럼 로그인된 상태라고 생각해봅시다. 하지만 사용자가 로그아웃되는 상황이 발생하기도 하죠. (브라우저가 닫히거나, DB가 재시작된다든가 등) 만약 로그인되지 않은 상태에서 새 글을 저장한다면, 사용자가 로그인되어있지 않아 누가 글을 작성하였는지 알 수 없어요. 그래서 글을 저장할 때 오류가 발생하고, 로그인시키도록 [http://127.0.0.1:8000/admin](http://127.0.0.1:8000/admin) 관리자 페이지가 나타나게 될 거에요. 이 문제는 금방 해결할 수 있어요. 튜토리얼을 마친 후 장고걸스 **심화 튜토리얼 - 보안 추가하기** 장에서 실습할 수 있어요.
```
Comment (soomin-jeong) [inline ko/django_forms/README.md]: I believe the contributor suggested to add a hyperlink to the "Django Girls Tutorial: Extensions". Please deal with this link!

File: ko/template_extending/README.md [hunk 5]
```diff
@@ -17,7 +17,8 @@
                 post_list.html
 
 
-그 다음 파일을 열어 `post_list.html`에 있는 모든 내용을 `base.html`에 아래 내용을 복사해 붙여넣습니다.
+그 다음 파일을 열어 `post_list.html`에 있는 모든 내용을 `base.html`에 아래 내용을 복사해 붙여넣습니다.  
+이렇게 말이죠:
```
Comment (soomin-jeong) [inline ko/template_extending/README.md]: It seems unnecessary.

File: ko/template_extending/README.md [hunk 6]
```diff
@@ -84,7 +85,7 @@
 이 코드는 무엇을 의미일까요? 우리는 `block`을 만든 거에요! 템플릿 태그 `{% block %}`으로 HTML 내에 들어갈 수 있는 공간을 만들었어요. `base.html`을 확장해 다른 템플릿에도 가져다 쓸 수 있게 한 거죠. 어떻게 동작하는지 보여 드릴게요.
 
 이제 `base.html`을 파일을 저장을 하고, `blog/templates/blog/post_list.html` 파일을 다시 엽니다. 
-{% raw %}`{% for post in posts %}`부터 `{% endfor %}`까지 내용 전체를 지우세요. 그럼 아래와 같이 될 겁니다. :{% endraw %}
+{% raw %}`{% for post in posts %}`의 위, 그리고 `{% endfor %}`아래의 모든 내용을 지우세요. 그럼 아래와 같이 될 겁니다. :{% endraw %}
```
Comment (soomin-jeong) [inline ko/template_extending/README.md]: A significant change to be updated as soon as possible.

**Comment by sujinleeme** [comment]
@corazzon Please check :)

**Comment by ekohl** [comment]
There is a merge conflict. This needs a rebase now.

**Comment by soomin-jeong** [comment]
Any suggestion for who to create the rebase?

**Comment by das-g** [comment]
The easiest is probably to make the improvements suggested here also on https://crowdin.com/project/django-girls-tutorial/ko where we manage the translations of the tutorial.

That will also avoid merge conflicts in the future, as we can then just adopt the text from there each time the translation is up-to-date and complete on Crowdin.

I'm closing this issue for now.

**Pull request closed without merging**